### PR TITLE
miscellaneous-changes

### LINKED ISSUE #162: miscellaneous  changes

### PR DESCRIPTION
**Pull request by 2016DigiTech**
miscellaneous changes all website. 

Fix #162 

Test URLs: https://miscellaneous-styling-changes--tlcgrp--hlxsites.hlx.page/
Test URLs: https://miscellaneous-styling-changes--tlcgrp--hlxsites.hlx.page/memberships-subscriptions
Test URLs: https://miscellaneous-styling-changes--tlcgrp--hlxsites.hlx.page/delivery
Test URLs: https://miscellaneous-styling-changes--tlcgrp--hlxsites.hlx.page/experience-vouchers
Test URLs: https://miscellaneous-styling-changes--tlcgrp--hlxsites.hlx.page/table-reservations

- Before: https://main--tlcgrp--hlxsites.hlx.page/
- After: https://miscellaneous-styling-changes--tlcgrp--hlxsites.hlx.page/

### CODE REVIEW

**Review by vdua — APPROVED** [review]

**Comment by 2016DigiTech** [comment]
Thanks for Approval (#163) Please  you don't merge I will merge only when management team approves.

**Pull request merged**
